### DISCUSSION TITLE
is it gonna be updated to new versions of minecraft?

### DISCUSSION THREAD

**Discussion by Justin7654** [CosmicHikari/Weather2-Remastered#15] [Questions] [answered]
i just want to know if your going to update it to 1.16+, but also support 1.12 so current worlds dont need to be restarted

**Comment by Mrbt0907** [comment, marked as answer]
I don't have any plans to update past existing versions of minecraft just yet. I am not capable of replacing coroutil for this mod to work in newer versions of minecraft like 1.16.

Things may change in the future though.

**Reply by CosmicHikari** [reply]
Coroutil 1.16.5 is available on curseforge, hopefully that can work with a future 1.16 version ^^
